cpp/cppsdk/gsdk.cpp

```diff
@@ -4,22 +4,34 @@
 #include "gsdkCommonPch.h"
 #include "gsdkInternal.h"
 #include "gsdkConfig.h"
+#include <stdio.h>
+#include <assert.h>
 
 namespace Microsoft
 {
     namespace Azure
     {
         namespace Gaming
         {
-            std::unique_ptr<GSDKInternal> GSDKInternal::m_instance = nullptr;
-            std::mutex GSDKInternal::m_gsdkInitMutex;
+            GSDKInternal* GSDKInternal::m_instance = nullptr;
             volatile long long GSDKInternal::m_exitStatus = 0;
             std::mutex GSDKInternal::m_logLock;
             std::ofstream GSDKInternal::m_logFile;
             bool GSDKInternal::m_debug = false;
             std::unique_ptr<Configuration> GSDKInternal::testConfiguration = nullptr;
 
             GSDKInternal::GSDKInternal() : m_transitionToActiveEvent(), m_signalHeartbeatEvent(), m_initialPlayers()
+            {
+
+            }
+
+			GSDKInternal::~GSDKInternal()
```

Comment:
NIT: Please use 4-spaces instead of tab-chars. For one, it makes this review almost impossible to view here in GitHub, but for 2, this file seems to already use 4-spaces everywhere.